### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by parvez-rashid**
This is the product listing page, by mistake the repo is named home-page.

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+:root {
+  --color-light-grey: #e2e8f0;
+}
+
+.product-display {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 1.1rem;
```
Comment (Sahil25061999) [inline styles/main.css]: use repeat here

**Review by Sahil25061999 — APPROVED** [review]
Not much code related suggestion.
In terms of UI i will suggest to use different font color for filter section since the contrast doesn't look great. Also add spacing between the heart icon and the text.
The positon of badge can be different by thats just my personal opinion, everything else looks good to me.

File: styles/main.css (hunk 1 above)
Comment (parvez-rashid) [reply styles/main.css]: Okay, will do that.

**Review by Bibin-Antony — APPROVED** [review]
The design seems like it needs work, other than that everything else seems ok.

**Pull request merged**